FirebaseMessaging/Sources/FIRMessaging.m

```diff
@@ -249,6 +246,57 @@ + (void)load {
   return @[ messagingProvider ];
 }
 
+- (void)configureMessagingWithOptions:(FIROptions *)options {
+  NSString *GCMSenderID = options.GCMSenderID;
+  if (!GCMSenderID.length) {
+    FIRMessagingLoggerError(kFIRMessagingMessageCodeFIRApp000,
+                            @"Firebase not set up correctly, nil or empty senderID.");
+    [NSException raise:kFIRMessagingDomain
+                format:@"Could not configure Firebase InstanceID. GCMSenderID must not be nil or "
```

Comment:
Installations or FCM token manager instead of InstanceID?

Reply:
Done.